docs/api/wrapper/isVueInstance.md

```diff
@@ -1,7 +1,11 @@
 ## isVueInstance
 
-::: warning
+::: warning Deprecation warning
 `isVueInstance` is deprecated and will be removed in future releases.
+
+Most of the times, tests relying on `isVueInstance` provide little to no-value so you could simply remove them.
```

Comment:
```suggestion
Tests relying on the `isVueInstance` assertion provide little to no value. We suggest replacing them with better assertions.
```

Active voice

Reply:
Even if you don't take the suggestion, "Most of the times" should be "Most of the time"